purity/gc/gc.go

```diff
@@ -187,6 +190,7 @@ func (c *collector) LoadAllBlobs() error {
 			for j := 0; j < 16; j++ {
 				ctx, cancel := context.WithTimeout(context.Background(), 30*time.Minute)
 				defer cancel()
+				ctx = metadata.NewOutgoingContext(ctx, metadata.Pairs("mettle-caller-name", name))
```

Comment:
I'm not normally that person, but maybe we should have a constant for `mettle-caller-name`...

Or possibly a top-level variable for this `Pairs` instance might be easier?

Reply:
Was considering moving it into the common package and perhaps making it a gRPC interceptor.

Not sure what conventions gRPC has for naming metadata as well, almost went for mettle.tmachine.io/name, but that feels very Java/k8s.

Reply:
> Keys are case insensitive and consist of ASCII letters, digits, and special characters -, _, . and must not start with grpc- (which is reserved for gRPC itself). Binary-valued keys end in -bin while ASCII-valued keys do not.

So you can't have the / at least. Lower kebab case seems normal from what I've seen.